src/python/requirements-app.txt

```diff
@@ -1,3 +1,5 @@
 ###
 ### Use this file for your custom view requirements here vs in the requirements-system.txt
-###
\ No newline at end of file
+###
+umap-learn
+dirty_cat
```

Comment:
Can we remove these? We were thinking of landing as part of this week's release and I saw this is lingering